react/props_and_component_structure/README.md

```diff
@@ -89,188 +84,53 @@ render() {
 }
 ```
 
-First, we're breaking out `products` from our state. Then, we're creating a variable called `listItems`, which maps through our `products` part of state and returns... React components called `ProductItem`.
+First, we're breaking out `products` from our class instance. Then, we're creating a variable called `listItems`, which maps through our `products` and returns React components named `ProductItem`.
 
-These components have certain attributes, but at first glance, they don't really make sense to us - there's no `className`, no `onChange`, nothing that we've really used before. That's because, **while these items share the same syntax, they don't translate directly to HTML attributes.** Instead, they represent *props* that we're passing down to our `ProductItem` component.
+These `ProductItem` components have three attributes that correspond to the keys of the objects in our Products array. At first glance, however, these attributes might look confusing. They definitely aren't anything we've seen before in HTML. That's because, **while these items share the same syntax, they don't translate directly to HTML attributes.** Instead, they represent *props* that we're passing down to our `ProductItem` components.
 
-Think of props as arguments for a component. They compile into an object called `props`, which the child component can key into and use to render stuff. In this case, for each item in our `products` part of state, we're inputting the name, manufacturer, and price into our `ProductItem` component. In fact, you can literally write a component that doesn't need to store state **as a function**. We call this a functional component, and `ProductItem` is our first example.
+Think of props as arguments for a component. They compile into an attribute on the component which we can reference using `this.props` (in the case of a class component) or as our function's argument (for a functional component). In this case, for each item in our `products` array, we're inputting the name, manufacturer, and price into our `ProductItem` component. You'll notice, because this component is only responsible for receiving props and rendering data, we're making this one functional, rather than using a class:
 
 ### `ProductItem.js`
 
 ```js
 import React from "react";
 
 const ProductItem = props => {
+  const { name, manufacturer, price } = props;
   return (
     <li>
-      {props.name} - {props.manufacturer} - ${props.price}
+      {name} - {manufacturer} - ${price}
```

Comment:
Suggestion: Let's have the initial example (shown here) only pass `name` and `price` as props. Then add a hands-on group / pairs exercise for fellows where they also pass through the manufacturer.